content/docs/user-guide/components/reference/_index.md

```diff
@@ -174,6 +174,7 @@ The following components are grouped by type as they appear in the O3DE Editor.
 | [PhysX Hinge Joint](/docs/user-guide/components/reference/physx/hinge-joint/) | Creates a dynamic hinge joint that constrains an entity to the joint with freedom to rotate around the x-axis of the joint.|
 | [PhysX Mesh Collider](./physx/mesh-collider/) | Allows you to specify PhysX mesh assets to calculate collisions between entities. |
 | [PhysX Primitive Collider](./physx/collider/) | Allows you to specify primitive shapes to calculate collisions between entities. |
+| [PhysX Prismatic Joint](/docs/user-guide/components/reference/physx/prismatic-joint/) | Creates a dynamic prismatic joint that constrains an entity to the joint keeping the same rotation but allowing to move freely along one axis.|
```

Comment:
nit: `Creates a dynamic prismatic joint that constrains an entity to the joint keeping the same rotation but allowing to move freely along one axis.` -> `Creates a dynamic prismatic joint that constrains an entity to the joint, keeping the same rotation but allowing it to move freely along one axis.`

Reply:
Thanks!